src/Controller/CampaignMonitorController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\campaignmonitor\Controller\CampaignMonitorController.
+ */
+
+namespace Drupal\campaignmonitor\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Form\FormBuilder;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Url;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Drupal\campaignmonitor\CampaignMonitor;
+
+class CampaignMonitorController extends ControllerBase {
+
+  /**
+   * The campaign monitor.
+   *
+   * @var \Drupal\campaignmonitor\CampaignMonitor
+   */
+  protected $campaignMonitor;
+
+  /**
+   * Settings for the module.
+   */
+  protected $settings;
+
+  /**
+   * Constructs a new SubscribeForm.
+   */
+  public function __construct() {
+    $this->campaignMonitor = CampaignMonitor::GetConnector();
+    $this->settings = \Drupal::config('campaignmonitor.general')->get();
```

Comment:
Watch out for calling things from the global namespace. You should be injecting the services `config.manager` and `form_builder` into this namespace and then including them as methods during create().

Use drupal console to bootstrap this controller with those services. I'll help you to avoid calling functions from the global \Drupal namespace.